onadata/libs/utils/analytics.py

```diff
@@ -172,12 +207,11 @@ def decorator(obj, *args):
 def track(user, event_name, properties=None, context=None, request=None):
     """Record actions with the track() API call to the analytics agents."""
     if _segment or appoptics_api:
-        context = context or {}
-        context['source'] = settings.HOSTNAME
```

Comment:
Why are we no longer tracking the `source`?

Reply:
It appears with the `campaign` [field](https://github.com/onaio/onadata/pull/1895/files#diff-9ddcf8387d802e0eed517864f979f2ff08dd99dbfabf287382548c1ef5771bfeR228) that segment configured.

Reply:
There is no `source` field known to Segment. The field to best represent this information is in the `campaign` object that we defined [here](https://github.com/onaio/onadata/pull/1895/files#diff-9ddcf8387d802e0eed517864f979f2ff08dd99dbfabf287382548c1ef5771bfeR194)